### ISSUE TITLE
No handler found for the command: 'extension.ftpsyncinit'. An extension might be missing an activation event.

### ISSUE THREAD

**Issue by Tigeraus** [lukasz-wronski/vscode-ftp-sync#60]
I use `init` command to initilize the ftp setting. But It shows 
`No handler found for the command: 'extension.ftpsyncinit'. An extension might be missing an activation event.`

**Comment by massimocode** [comment]
Hey. I've found this is usually an issue with VS Code extensions when you first install/update them. Try restarting your editor and see if the issue persists. If it does then let us know what version of VS Code and what version of the extension (and your OS etc) and we'll look into it.

**Comment by felippeDev** [comment]
@massimocode Yes... i've restarted my VSCode and the issue persists. I'm using the latest version (until now) OSX Version 1.3.1 (1.3.1) - e6b4afa53e9c0f54edef1673de9001e9f0f547ae
